raysect/core/math/quaternion.pyx

```diff
@@ -0,0 +1,556 @@
+
+import numbers
+cimport cython
+from libc.math cimport sqrt, sin, cos, M_PI
+
+from raysect.core.math.affinematrix cimport new_affinematrix3d
+
+
+cdef double DEG2RAD = 2*M_PI / 360
```

Comment:
Copy the DEF lines from math/transform.pyx, they are precalculated.

I'll make a central constants file at some point.

Reply:
Resolved in a4d9d8a